CONTRIBUTING.md

```diff
@@ -3,13 +3,29 @@ Contributing
 
 It's good to hear that you want to contribute to Pony!
 
-First of all please [search existing issues][complete-issue-list]. Apart from bug reports you will find feature requests and work currently in progress too. If you cannot find a suitable issue — [create a new one][new-issue]. Potential pull requests are no exception. Before you start working, let's have a discussion and agree on the solution and scope.
+There are a number of ways to contribute to Pony. As this document is a little long, feel free to jump to the section that applies to you currently:
 
-Each of [bug report](#bug-report), [feature request](#feature-request) or [pull request](#pull-request)
-has to cover different aspects described below.
+* [Feature request](#feature-request)
+* [Bug report](#bug-report)
+* [Pull request](#pull-request)
+
+Additional notes regarding formatting:
+
+* [Documentation formatting](#documentation-formatting)
+* [Code formatting](#code-formatting)
+
+Feature request
+---------------
+To request a change to the way that Pony works, please open an issue in the [RFCs repository](https://github.com/ponylang/rfcs) rather than this one. New features and other significant language changes must go through the RFC process.
+
+If you submit a pull request to implement a new feature without going
+through the RFC process, it may be closed with a polite request to
+submit an RFC first.
```

Comment:
i just noticed the bunny @jemc. This bunny is doubly cute in gmail with its different emoji. Either way, I just got a big smile. More bunnies in the future please.